src/app/control.rs

```diff
@@ -263,10 +268,10 @@ pub mod resolve {
             let target = client::Target {
                 addr,
                 server_name: dst.identity.clone(),
-                log_ctx: logging::admin().client("control", dst.addr.clone()),
             };
 
-            State::Inner(mk_svc.call(target))
+            info_span!("control", peer_addr=%addr, peer_identity=?dst.identity.clone())
```

Comment:
Cloning the identity shouldn't be necessary — it's recorded as a reference
```suggestion
            info_span!("control", peer_addr=%addr, peer_identity=?dst.identity)
```

Reply:
Nit/style: Typically prefer spaces around the `=` in `tracing` macros, the way you'd format a Rust assignment.
```suggestion
            info_span!("control", peer_addr = %addr, peer_identity = ?dst.identity.clone())
```

Reply:
Nit/TIOLI: you can also namespace `tracing` field names with dots, so this could be
```suggestion
            info_span!("control", peer.addr=%addr, peer.identity=?dst.identity.clone())
```
future `tracing` APIs will have nice globbing support for field names based on dots.

Reply:
i suppose rustfmt has trouble with the proc-macro...

Reply:
it's actually just a `macro_rules!` macro; AFAICT rustfmt doesn't really touch anything inside macro invocations